<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>650</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>755</width>
    <height>479</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>1024</width>
    <height>650</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="HeadWidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1024</width>
      <height>142</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>1024</width>
      <height>142</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>1024</width>
      <height>142</height>
     </size>
    </property>
   </widget>
   <widget class="QWidget" name="TabsWidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>145</y>
      <width>167</width>
      <height>289</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="TagsWidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>440</y>
      <width>167</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="searchButton">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>145</y>
      <width>95</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="filterButton">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>145</y>
      <width>126</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteButton">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>145</y>
      <width>126</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QWidget" name="foldersWidget" native="true">
    <property name="geometry">
     <rect>
      <x>169</x>
      <y>210</y>
      <width>851</width>
      <height>53</height>
     </rect>
    </property>
    <widget class="QPushButton" name="addFolderButton">
     <property name="geometry">
      <rect>
       <x>730</x>
       <y>0</y>
       <width>121</width>
       <height>53</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="searchString">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>145</y>
      <width>461</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
